docs/src/guides/wordpress/composer/migrate.md

```diff
@@ -1,112 +1,136 @@
 ---
-title: "How to update your WordPress site to use Composer"
+title: "How to upgrade your WordPress site to use Composer"
 sidebarTitle: Upgrade to use Composer
 weight: 2
 description: |
-    Everything you need to get started with WordPress on Platform.sh. 
+    Learn how to use WordPress with Composer on Platform.sh.
 ---
 
-This guide will take you through the steps to update your [*vanilla*](/guides/wordpress/vanilla/_index.md), fully committed, WordPress repository to one fully managed with Composer. It is assumed that you already have (locally) a vanilla version of WordPress where all of core, themes, and plugins are committed to the repository or committed as submodules, and that it has been set up for deployment on Platform.sh. Because of these assumptions, you should already have Platform.sh configuration files in your repository to deploy the resulting Composer-based WordPress site on Platform.sh. If you do not, follow the [previous guide](/guides/wordpress/vanilla/_index.md) and add them. 
+Composer helps you declare, manage, and install all the dependencies needed to run your project.
+It allows you to make your WordPress site [more stable, more secure, and easier to maintain](./_index.md).
 
-For more context regarding why Platform.sh recommends Composer-based installations of WordPress, see the ["Using Composer" guide](/guides/wordpress/composer/_index.md).
+To update your WordPress site to use Composer, check that:
 
-## WordPress Core
+- You already have a [vanilla version of WordPress installed locally](../vanilla/_index.md).
+- Your project has been set up for deployment on Platform.sh.
+  If you don't have Platform.sh configuration files in your repository,
+  [deploy WordPress without Composer](../vanilla/_index.md) before upgrading to a Composer-based site.
+- You have [downloaded and installed Composer](https://getcomposer.org/download/).
 
-Instead of committing all of WordPress to your repository (or adding it as a submodule), this section will show you how to install WordPress through Composer. 
+## Install WordPress with Composer
 
-1. **Download Composer** 
+Managing WordPress with Composer makes installing and updating WordPress and its dependencies much more lightweight.
+With Composer, you don't need to commit all of WordPress core, its themes and plugins to your project's Git repository. 
+You also don't need to manage any of these elements as Git submodules. 
 
-    If you do not already have Composer installed, [install it now](https://getcomposer.org/download/).
+If you were managing WordPress and its dependencies as Git submodules, 
+see [how to remove them](../../../development/submodules.md#removing-submodules) before switching to Composer.
 
```

Comment:
My thinking at the time was that since the use of Composer is a recommended alternative to Git submodules, maybe we should make it clear from the beginning of the procedure (or even in the prerequisites, only it wouldn't apply to everyone so I didn't really want to go for that) that Git submodules should be removed from the get go. But moving it to the start of step 3 seems to work, let me know if it's ok the way I just changed it.